src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -273,12 +286,21 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
                     // eslint-disable-next-line no-console
                     console.warn(`Imagery texture update error for ${node}: ${err}`);
                 }
-                const definitiveError = node.layerUpdateState[layer.id].errorCount > MAX_RETRY;
-                node.layerUpdateState[layer.id].failure(Date.now(), definitiveError);
-                if (!definitiveError) {
-                    window.setTimeout(() => {
-                        context.view.notifyChange(false, node);
-                    }, node.layerUpdateState[layer.id].secondsUntilNextTry() * 1000);
+                const nextLevelToTry = targetLevel - 1;
+                const pFailureParams = node.layerUpdateState[layer.id].failureParams;
+                const pFailureLevel = pFailureParams ? pFailureParams.targetLevel : undefined;
+                // notify a new try specifying failure parameters
+                if (nextLevelToTry != pFailureLevel) {
```

Comment:
I'm not sure to understand the code nor the comment. Can you explain a bit more?

Reply:
I'll continue to search because without `layer.tileInsideLimit (node, layer, nextLevelToTry)`, the test no longer has the same meaning, and may be the same behaviour.
I'll also clarify a comment